docs/reference/data-operations.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[data-operations]]
+= Roll up or transform your data
```

Comment:
Since this is the name of the top-level page, I think it would be better to be more verbose for SEO purposes. Maybe `[data-rollup-transform]`?